english/super-man/save-the-word.md

```diff
@@ -15,3 +15,7 @@ They are all making a unique voice:  calling for a Man like you! Do you:
 [Find the evil mastermind Lex Luther?](./lex-luther/find-lex-luther.md)
 
 [Find the bird?](../find-bird/bird.md)
+
+Or do you rid those decisions from your mind and decide that you want to be a new "Super" character all together! You're not a one role kinda guy, you don't even fit into the DC universe!! You decide you must become... 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.